core/common/src/main/java/alluxio/Constants.java

```diff
@@ -158,6 +158,10 @@
       "alluxio.underfs.s3.disable.dns.buckets";
   public static final String UNDERFS_S3_SERVER_SIDE_ENCRYPTION =
       "alluxio.underfs.s3.server.side.encryption";
+  public static final String UNDERFS_S3_MAX_THREADS = "alluxio.underfs.s3.threads.max";
```

Comment:
could you change your constant name to be `UNDERFS_S3_THREADS_MAX` to be consistent ?